.chloggen/add_inspect_metrics.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: dockerstatsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add [container.cpu.limit], [container.cpu.shares] and [container.restarts] metrics from docker container api"
+
+# One or more tracking issues related to the change
+issues: [21087]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: It requires API version to 1.25 or greater if the new metrics are enabled.
```

This isn't conditionally true since the api version check is enforced in config validation.

---

That's right. We may move it back to breaking or stop enforcing the api version check. WDYT?

---

I'm referring to the language in this statement, which isn't accurate.

```suggestion
subtext: It requires API version to 1.25 or greater.
```